### PR TITLE
Html report

### PR DESCRIPTION
**Pull request by bburns632**
Rebased version of #36

### CODE REVIEW

**Comment by bburns632** [comment]
Also, tests fail on this branch, but it's a start

File: R/CreatePackageReport.R [hunk 1]
```diff
@@ -45,5 +46,35 @@ CreatePackageReport <- function(packageName
...
+
+#' @title Package Report Renderer
+#' @name RenderPackageReport
+#' @description Renders an html report based on the initialized reporters provided
+#' @author P. Boueri
+#' @param reportPath a file.path to where the report should be rendered
+#' @param packageReporters a list of package reporters that have already been initialized and have calculated 
+#' @return Nothing
+RenderPackageReport <- function(reportPath 
+                                , packageReporters) {
```
Comment (jameslamb) [inline R/CreatePackageReport.R]: this should have a default of `DefaultReporters()`

File: R/PackageSummaryReporter.R [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+#' @importFrom R6 R6Class
+#' @export
+PackageSummaryReporter <- R6::R6Class(
+    classname = "PackageSummaryReporter",
+    inherit = AbstractPackageReporter,
+    public = list(
+        get_report_markdown_path = function(){
+            system.file(file.path("package_report","package_summary_reporter.Rmd"),package = "pkgnet")
+        },
+        get_description = function(){
```
Comment (jameslamb) [inline R/PackageSummaryReporter.R]: these getters should be active bindings

Comment (bburns632) [reply R/PackageSummaryReporter.R]: why?

**Pull request merged**